pkg/netdevice/netResourcePool.go

```diff
@@ -80,3 +84,42 @@ func (rp *netResourcePool) GetDeviceSpecs(deviceIDs []string) []*pluginapi.Devic
 	}
 	return devSpecs
 }
+
+// StoreDeviceInfoFile stores the Device Info files according to the
+//  k8snetworkplumbingwg/device-info-spec
+func (rp *netResourcePool) StoreDeviceInfoFile(resourceNamePrefix string) error {
+	for id, dev := range rp.GetDevicePool() {
+		netDev, ok := dev.(types.PciNetDevice)
+		if !ok {
+			return fmt.Errorf("StoreDeviceInfoFile: Only pciNetDevices are supported")
+		}
+		devInfo := nadtypes.DeviceInfo{
+			Type:    "pci",
+			Version: "0.1.0",
```

Comment:
I added a `nettypes.DeviceInfoVersion` so that it always updates the version based on what version of network-attachment-definition-client the code is compiling against.

Reply:
+1 on using defined const.

Reply:
Done